docs/docs/06-cli.md

```diff
@@ -4,48 +4,73 @@ title: MemoryViz CLI
 
 # MemoryViz CLI
 
-You can run MemoryViz straight from your terminal!
+Run MemoryViz straight from your terminal!
 
-## Input
+The MemoryViz CLI takes in MemoryViz-compatible JSON and returns an SVG of the memory model.
 
-To run the MemoryViz CLI, run:
+## Usage
+
+To use the MemoryViz CLI, run:
 
 ```console
 $ npx memory-viz <path-to-file>
 ```
 
-replacing `<path-to-file>` with the path to a file containing MemoryViz-compatible JSON. If the file content is not compatible with MemoryViz, an error will be thrown.
+where `<path-to-file>` is the path to a file containing MemoryViz-compatible JSON. If a file path is not provided, the CLI will take input from standard input.
+
+By default, the SVG is saved in the current working directory. See the "Options" section below for more output options.
 
-## Output
+NOTE: If you choose not to provide a file path and are typing JSON directly into the terminal, you need to signal the end of your input.
 
-The output is an SVG image generated by MemoryViz and the image is saved in the current working directory. The name of the SVG will be the same as that of the inputted file (i.e., if the inputted file is `david-is-cool.json`, the output will be `david-is-cool.svg`).
+-   **macOS, Linux:** `Ctrl + D`
+-   **Windows:** `Enter` followed by either
```

Comment:
This isn't correct, only `Ctrl + D` is needed. See my comment below about how to read in the input data.